mlir/test/Target/LLVMIR/llvmir-intrinsics.mlir

```diff
@@ -1111,6 +1111,35 @@ llvm.func @vector_ptrmask(%p: !llvm.vec<8 x ptr>, %mask: vector<8 x i64>) -> !ll
   llvm.return %0 : !llvm.vec<8 x ptr>
 }
 
+// CHECK-LABEL: @experimental_constrained_uitofp
+llvm.func @experimental_constrained_uitofp(%s: i32, %v: vector<4 x i32>) {
+  // CHECK: call float @llvm.experimental.constrained.uitofp.f32.i32(
+  // CHECK: metadata !"round.towardzero"
+  // CHECK: metadata !"fpexcept.ignore"
+  %0 = llvm.intr.experimental.constrained.uitofp %s towardzero ignore : i32 to f32
```

Comment:
Note that checking all combinations here is not really necessary since the logic is already tested by sitofp and probably other operations.

Reply:
OK, I'll take into account when implementing next intrinsics.